test/CMakeLists.txt

```diff
@@ -1,16 +1,14 @@
 add_subdirectory(data)
 
 # add_grail_executable([BINNAME <binary name>] SRC <src path> LIBS <libs>)
-add_grail_executable(SRC ESRIMapDemo.cc LIBS grail)
+#add_grail_executable(SRC ESRIMapDemo.cc LIBS grail)
```

Comment:
Currently ESRIMapDemo is excluded from the test build. This will be fixed later, but ESRIMapDemo currently has too many callbacks for this to be a quick fix. This will probably be fixed in a separate commit.